src/assets/scripts/client/aircraft/AircraftController.js

```diff
@@ -618,11 +652,29 @@ export default class AircraftController {
             icao: aircraftTypeDefinition.icao,
             model: aircraftTypeDefinition,
             routeString: spawnPatternModel.routeString,
+            commands: spawnPatternModel.commands,
             // TODO: this may not be needed anymore
             waypoints: _get(spawnPatternModel, 'waypoints', [])
         };
     }
 
+    /**
+     * Execute a command on a new or preSpawned aircraft, ignoring controllability flag
+     *
+     * @param aircraft {AircraftModel}
+     * @param command {String}
+     * @private
+     */
+    _runCommandOnNewAircraft(aircraft, commandString) {
```

Comment:
`_runCommandOnNewAircraft` -> `_runCommandOnPreSpawnAircraft`

There is some logic above that looks to be duplicated.  What if we called this method for _every_ prespawn aircraft?  If an aircraft doesn't have commands to run, then we return early.  If they do, we proceed and run commands

Reply:
renamed and changed to return-early style